perf/src/sigverify.rs

```diff
@@ -773,13 +816,22 @@ mod tests {
             }
 
             // verify packets
-            let ans = sigverify::ed25519_verify(&batches, &recycler, &recycler_out);
-
-            let cpu_ref = ed25519_verify_cpu(&batches);
+            let ans = sigverify::ed25519_verify(&mut batches, &recycler, &recycler_out);
+            ed25519_verify_cpu(&mut batches);
```

Comment:
I think this test intended to isolate the behavior of `ed25519_verify_cpu` against `sigverify::ed25519_verify` but now that `batches` is passed by mutable reference, this is no longer the case. Feels like batches should be cloned into `sigverify::ed25519_verify` to prevent that.

Reply:
hehe yeah, updated to do that!